### PR TITLE
Minor updates for 0.9.1 release

### PR DESCRIPTION
**Pull request by co9olguy**
- Both of the native gates have a grad recipe which fits the PennyLane default. [see here](https://www.aqt.eu/aqt-gate-definitions/)
- Lowered the retry timer so the plugin won't hammer the server for results
- removed an unnecessary import

@josh146 Should be good to go for a v0.9.1 release after this PR is merged

### CODE REVIEW

File: pennylane_aqt/device.py [hunk 1]
```diff
@@ -103,7 +102,7 @@ def set_api_configs(self):
             raise ValueError("No valid api key for AQT platform found.")
         self.header = {"Ocp-Apim-Subscription-Key": self._api_key, "SDK": "pennylane"}
         self.data = {"access_token": self._api_key, "no_qubits": self.num_wires}
-        self.hostname = urllib.parse.urljoin("{}/".format(self.BASE_HOSTNAME), self.TARGET_PATH)
```
Comment (co9olguy) [inline pennylane_aqt/device.py:106]: This was bothering me, so I removed

File: pennylane_aqt/device.py [hunk 2]
```diff
@@ -78,7 +77,7 @@ class AQTDevice(QubitDevice):
     TARGET_PATH = ""
     HTTP_METHOD = "PUT"
 
-    def __init__(self, wires, shots=BASE_SHOTS, api_key=None, retry_delay=0.05):
+    def __init__(self, wires, shots=BASE_SHOTS, api_key=None, retry_delay=1):
```
Comment (co9olguy) [inline pennylane_aqt/device.py:80]: also adjusted this to avoid hammering server

File: pennylane_aqt/ops.py [hunk 3]
```diff
@@ -68,5 +67,4 @@ class MS(Operation):
     num_params = 1
     num_wires = 2
     par_domain = "R"
-    grad_method = None
-    grad_recipe = None
+    grad_method = "A"
```
Comment (josh146) [inline pennylane_aqt/ops.py:70]: Ah, so the MS gate supports the default parameter-shift in the end?

Comment (co9olguy) [reply pennylane_aqt/ops.py:70]: Looks like it, yes

**Pull request merged**